core/src/v2/actix.rs

```diff
@@ -277,6 +277,11 @@ impl OperationModifier for actix_multipart::Multipart {
     }
 }
 
+#[cfg(feature = "actix-session")]
+impl OperationModifier for actix_session::Session {
+    fn update_parameter(_op: &mut DefaultOperationRaw) {}
```

Comment:
We can get rid of this function, because this is the default implementation. We should override `update_definitions` instead and have an empty function body.

Reply:
Yep, sorry, missed that